app/Graph/build.hpp

```diff
@@ -33,6 +34,8 @@
 #include "layers/TransposeLayer.hpp"
 #include "layers_oneDNN/EWLayer.hpp"
 
+using namespace it_lab_ai;
+
```

Comment:
Please, do not add `using namespace ...` to the headers. This leads to the namespace "pollution" everywhere where `app/Graph/build.hpp` is included